### ISSUE TITLE
dxgi.dll not found/no log files

### ISSUE THREAD

**Issue by fgblomqvist** [doitsujin/dxvk#245]
### Software information
World of Tanks v1.0.0.2 with any settings

### System information
- GPU: NVIDIA 1060
- Driver: Proprietary (390)
- Wine version: 3.5 Staging
- DXVK version: 0.41

### Apitrace file(s)
Will generate one if this is actually related to the game.

### Log files
- d3d11.log: Not generated
- dxgi.log: Not generated

### Description
Neither of the two log files are created, which is odd because DXVK is definitely being used (the performance in the game is waaaay better with DXVK than without). I just used the setup script, which set up the two overrides. However, it seems like the dxgi override might not be working? The HUD doesn't work either.

I have attached the full log from Wine as well. Let me know if you need any other info. Others running WoT in Wine with dxvk have reported the same thing.

This is a line that appears quite a few times:
0030:err:module:import_dll Library dxgi.dll (which is needed by L"C:\\windows\\system32\\wbemprox.dll") not found

[wine-wot.log](https://github.com/doitsujin/dxvk/files/1883076/wine-wot.log)

As a side note, it seems to happen at the same time as the game freezes for 10-20 seconds. First on the loading screen, and then every time you go to the game's "lobby" (the "garage"). Not sure if it's related or not but figured I would mention it.

**Comment by doitsujin** [comment]
This sounds like a setup issue. If you look at the log files, it's pretty obvious that DXVK is *not* being used. Most of the messages are wined3d related.

Make sure that your wine installation actually supports Vulkan and that you have **both** the 64-bit and 32-bit DXVK DLLs set up.

**Comment by fgblomqvist** [comment]
@Kerrung All of those were checked (to clarify, the wineprefix needs to run a Windows mode that supports DX11, such as Windows 7, 10, etc.)

@doitsujin Something was wrong with my configs, and I also did not set it up for x32. I created a new prefix, set it up for 32 and 64 and then it worked great. Now the HUD and logs are working (as in, dxvk is actually used). I'm running 3.5 staging without the VulkanSDK. Thanks for the clarification!
